### ISSUE TITLE
[doc|AbstractRouteDef] AbstractRouteDef is not importable from aiohttp.web

### ISSUE THREAD

**Issue by Maillol** [aio-libs/aiohttp#3183]
## Long story short
Currently the documentation mentions `aiohttp.web.AbstractRouteDef` but `AbstractRouteDef` is not importable from aiohttp.web

http://docs.aiohttp.org/en/stable/web_reference.html?highlight=AbstractRouteDef#aiohttp.web.AbstractRouteDef
 
## Expected behaviour

`AbstractRouteDef` is importable from aiohttp.web
or documentation mentions `aiohttp.web_routedef. AbstractRouteDef`

## Actual behaviour

raise ImportError

## Steps to reproduce

```
>>> from aiohttp.web import AbstractRouteDef
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: cannot import name 'AbstractRouteDef'
```

## Your environment
aiohttp==3.3.2

**Comment by asvetlov** [comment]
It should be importable from `aiohttp.web`.
Would you make a pull request? The change is trivial.

I'm curious why do you need to import `AbstractRouteDef`?
From my perspective, it is a very low-level API. Users almost never need it.

**Comment by Maillol** [comment]
I will make a pull request this weekend.

I use a custom `AbstractRouteDef` to route multiple handlers define in the class.
I don't use web.View because the handlers of my classes is not mapped to same URL:
```
post --> /resource
get  --> /resource/{id}
get  --> /resource
put --> /resource/{id}/{action}
delete -->  /resource/{id}
```

**Comment by asvetlov** [comment]
Neat!

**Comment by asvetlov** [comment]
Fixed by #3188
